firmware/config/engines/honda_k_dbc.cpp

```diff
@@ -42,8 +43,11 @@ void setHondaK() {
 	engineConfiguration->injectionMode = IM_SEQUENTIAL;
 
 	engineConfiguration->crankingIACposition = 70;
-    engineConfiguration->postCrankingFactor = 1.25;
-    engineConfiguration->postCrankingDurationSec = 15;
+	static const uint16_t defaultPostCrankinDurationBins[] = {
```

Comment:
say honda K overrides what we have in setDefaultCranking()

meaning you already have axis, you only need to override table

Reply:
yes, the axis was changed, as Honda used here 15 instead of default 10 seconds

Reply:
we can either have different Z values in table, or change axis to reproduce what was done before